packages/react-dom/server-rendering-stub.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+// Export all exports so that they're available in tests.
+// We can't use export * from in Flow for some reason.
+export {default as __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED} from './src/ReactDOMSharedInternals';
```

Comment:
When we remove the legacy stuff from `react-dom/index` we'll still have a few exports remaining.

```
createPortal
flushSync
version
```

While you shouldn't call these functions, they would still exist as exports but not do anything. Maybe we can add empty stubs that warn if you call them.

`version` should probably just work.